From 3f6cb42d32dc5472a3ca3d68eae51f55c67c53c2 Mon Sep 17 00:00:00 2001
From: Wei-Chiu Chuang <weichiu@apache.org>
Date: Thu, 4 Aug 2016 19:40:33 -0700
Subject: [PATCH 1686/2848] HDFS-10588. False alarm in datanode log - ERROR -
 Disk Balancer is not enabled. Contributed by
 Weiwei Yang.

(cherry picked from commit 18d9e6ec0bdb4bce316f8af5d3f13902dd899325)

Change-Id: I69c44ab23a7dfdfcd9c30cd50fe29c1bc50254d8
---
 .../hadoop/hdfs/server/datanode/DiskBalancer.java  |    1 -
 1 file changed, 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DiskBalancer.java b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DiskBalancer.java
index 3903879..32c8b97 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DiskBalancer.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DiskBalancer.java
@@ -294,7 +294,6 @@ public long getBandwidth() throws DiskBalancerException {
   private void checkDiskBalancerEnabled()
       throws DiskBalancerException {
     if (!isDiskBalancerEnabled) {
-      LOG.error("Disk Balancer is not enabled.");
       throw new DiskBalancerException("Disk Balancer is not enabled.",
           DiskBalancerException.Result.DISK_BALANCER_NOT_ENABLED);
     }
-- 
1.7.9.5

